admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/websub/dto/Event.java

```diff
@@ -0,0 +1,16 @@
+package io.mosip.kernel.syncdata.websub.dto;
```

Comment:
do we not have an existing class in kernel for this? as this is a generic format of the event?